src/actions/tamanu/psql.rs

```diff
@@ -0,0 +1,140 @@
+use std::path::Path;
+use std::{fs, path::PathBuf};
+
+use clap::Parser;
+use miette::{miette, Context as _, IntoDiagnostic, Result};
+use tracing::{debug, info, instrument};
+
+use crate::actions::Context;
+
+use super::config::{merge_json, package_config};
+use super::{find_tamanu, TamanuArgs};
+
+/// Connect to Tamanu's db via `psql`.
+#[derive(Debug, Clone, Parser)]
+pub struct PsqlArgs {
+	/// Package to look at. By default, the command finds installed version automatically
+	/// If both central and facility server were present, there's no guarantee which server this picks
+	#[arg(short, long)]
+	pub package: Option<String>,
+
+	/// Include defaults
+	#[arg(short = 'D', long)]
+	pub defaults: bool,
+
+	/// If given, this overwrites the username in the config and prompts for a password
```

Comment:
```suggestion
	/// Connect to postgres with a different username.
	///
	/// This may prompt for a password depending on your local settings and pg_hba config.
```